acceptable/tests/test_main.py

```diff
@@ -393,11 +386,11 @@ def test_render_markdown_success(self):
 
             md = yaml.safe_load(output['en/metadata.yaml'])
             self.assertEqual({
-                    'navigation': [
-                        {'location': 'index.md', 'title': 'Index'},
-                        {'location': 'group.md', 'title': 'Group'},
-                    ],
-                },
+                'navigation': [
+                    {'location': 'index.md', 'title': 'Index'},
+                    {'location': 'group.md', 'title': 'Group'},
+                ],
+            },
```

Comment:
Most of the changes in this file are formatting, afaics. It'd have been great to move all of them to a separate commit.